### DISCUSSION TITLE
搭元宇宙熱潮 外貿協會年度會虛實同步 民視影音．466 次觀看．2 小時前

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#211] [General]
為了深化台美的經貿關係，這次也瞄準了電動車、醫療保健、運動樂活等五大關鍵產業布局，要提高台灣品牌的能見度。
https://tw.tv.yahoo.com/ftv/%E6%90%AD%E5%85%83%E5%AE%87%E5%AE%99%E7%86%B1%E6%BD%AE-%E5%A4%96%E8%B2%BF%E5%8D%94%E6%9C%83%E5%B9%B4%E5%BA%A6%E6%9C%83%E8%99%9B%E5%AF%A6%E5%90%8C%E6%AD%A5-041003339.html

**Comment by handsomeniko** [comment]
現在相關領域的確是炙手可熱，金流都往這邊。

**Comment by Shiba010** [comment]
元宇宙在今年真的是滿有話題性的，看起來也是未來的趨勢。

**Comment by yasklop** [comment]
最近有關元宇宙的話題好多，感覺大家需要多了解一下
